test/utils/setup.ts

```diff
@@ -20,7 +24,7 @@ type LogGas = {
     readonly logGasUsage?: boolean;
 };
 
-type SafeCreationWithGasLog = SafeWithSetupConfig & LogGas;
+type SafeWithCustomSingletonAndGasLog = SafeSingleton & SafeWithSetupConfig & LogGas;
```

Comment:
Nit: I would name this type:

```suggestion
type GetSafeParameters = SafeSingleton & SafeWithSetupConfig & LogGas;
```

Reply:
Updated 👍🏾